Rakefile

```diff
@@ -60,4 +60,9 @@ namespace :shipyard do
     icons = Shipyard::Icons.new args.icon_directory, args.output_directory
     icons.write
   end
+
+  desc 'Publish project to npm as a scoped package @codeship/shipyard'
+  task :console do
+    sh 'npm publish --access public'
```

Comment:
Two questions:
1. Does this need to authenticate with anything?
2. Do we need to add this command to the CI `deploy` stage?